control/control_env.py

```diff
@@ -57,3 +57,6 @@
 NODE_COLORS_GREEN = '\033[92m'
 NODE_COLORS_RED = '\033[91m'
 NODE_COLORS_END = '\033[0m'
+
+HEAL_START = 6
+HEAL_END = 144
```

Comment:
@yro It would be great if we can add a comment about why we choose these particular values?